tests/unit/core.js

```diff
@@ -872,6 +877,7 @@ exports.testConstModification = function (test) {
       .addError(8, "Attempting to override 'a' which is a constant.")
       .addError(8, "You might be leaking a variable (a) here.")
       .addError(53, "Missing '()' invoking a constructor.")
+      // .addError(17, "Attempting to override 'f' which is a constant.") TODO
```

Comment:
okay, If I don't fix it in this Pr I'll do that.
Line no is a mistake due to merging